.circleci/config.yml

```diff
@@ -221,23 +242,39 @@ jobs:
           command: |
             aws s3 sync "s3://login-gov-test-coverage/coverage/$CIRCLE_BUILD_NUM" coverage/
             ./cc-test-reporter sum-coverage --output - --parts $CIRCLE_NODE_TOTAL coverage/codeclimate.*.json | ./cc-test-reporter upload-coverage --input -
-  javascript_build:
+  javascript_test:
     working_directory: ~/identity-idp
     executor: ruby_browsers
     steps:
       - checkout
-      - bundle-yarn-install
+      - node-install
+      - yarn-install
       - run:
           name: Run Tests
           command: |
             yarn test
 
+  javascript_build:
```

Comment:
I like the name here as far as verifying that we can actually build assets, but it leaves me a bit funny that we have the unprefixed "build" job as meaning something totally different, where it's more about running RSpec tests.

Reply:
Agreed, changed the `build` job to be named `ruby_test`